### ISSUE TITLE
committed typo in rungeneric1

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco-postprocess#25]
https://github.com/numbbo/coco-postprocess/blob/0e182efa2dde21ee29500c3b52a943e34a9f16c1/src/cocopp/rungeneric1.py#L102

`inset` was supposed to refer to something else, e.g. `genericsettings`, but what was it?

**Comment by nikohansen** [comment]
@brockho , do you have an idea?

**Comment by brockho** [comment]
Indeed, in older versions of `rungeneric1.py`, I found `from . import genericsettings as inset`. Most likely, line 102 was simply not updated, when the import was removed.
